README.rst

```diff
@@ -168,6 +168,41 @@ Remove a container::
 
     courseraresearchexports containers remove $CONTAINER_NAME_OR_ID
 
+db
+^^
+
+create_view
+~~~~~~~~~~~
+Create a view in the dockerized database. We are planning to include commonly
+used denormalized views as part of this project. To create one of these views
+(i.e. for the demographic_survey view)::
+
+    courseraresearchexports db create_view $CONTAINER_NAME_OR_ID --view_name demographic_survey --partner_short_name $PARTNER_SHORT_NAME
```

Comment:
There's two options:
1) Expose the partner_id that's returned with `ExportRequestWithMetadta` and use that for inference.
2) Find the column name that matches `user_id` in the `users` table.